test/helper.js

```diff
@@ -104,6 +104,11 @@ function setup(file, codeArray, opts) {
                 }
                 return;
             }
+
+            // `export`/`import` cannot be wrapped inside a function.
+            // For our purposes, simply remove the `export` from export declarations.
+            generated = generated.replace(/export (var|function|let|const)/g, '$1');
```

Comment:
This should only be done in the `opts.esModules: true` case. Hacky code like this should be reduced to as small a footprint as possible!